fastlane/screenshots/README.md

```diff
@@ -0,0 +1,129 @@
+ - Wait for the release manager to do the next code freeze, which will include the new content of those `.txt` files into the `PlayStoreStrings.po` file and then will be imported into GlotPress for translation (same process than for the release notes translation)
+ - During the beta test phase that runs post code freeze, polyglots will then translate your new copies into the various locales in GlotPress.
+ - At the end of the sprint, once all the translations have been done and retrieved from GlotPress as part of the release finalization, you should then have `fastlane/{jetpack_,}metadata/android/{locale}/screenshot_{n}.txt` files translated in each of the locales, ready to be referenced by your config file and used for the promo screenshots generation in all the locales.
+
+If you need a faster feedback loop or to accelerate the translation, get in touch with your Release Manager and/or with Team Global, to see if we can fasttrack the translations outside of the regular cycle; though polyglots will still always need time to do translations, so keep in mind that this will never be instantaneous and you need to account for it in your planning about updating PlayStore screenshots.
+
+### Installing `imagemagick` and `rmagick`
+
+To do the image compositing, we rely on the ImageMagick library and the Ruby wrapper gem `rmagick` that interfaces it.
+This means that you will need to install those on your machine first before being able to run the lane generating the promo screenshots.
+
+ - Run `brew install imagemagick` to install the ImageMagick library on your system
+ - Then run `bundle install --with screenshots` to include the `rmagick` gem in the list of gems installed.
+    - This will make a change to the `.bundle/config` file of the repository. Please don't commit that change, so that other developers won't require to have `rmagick` installed for everything else they work on — as this library and gem is only needed for generating screenshots, which is not as common as all the other everyday tasks developers run.
+
+> **Note**: This (especially `brew install imagemagick`) requires you to install something on you machine system-wide (`brew install` install things at the machine level, for all users, unlike `bundle` which installs things only in the folder of your current repo / working copy). So this is a bit invasive.
+>
+> This also sadly comes with risks of having installation troubles depending on the environment of your machine (and which libraries and headers you have installed, etc); in fact, it's sadly not uncommon to have issues when trying to `brew install imagemagick` or to install `rmagick`, especially on the steps which tries to compile those libraries (since they rely on compiled, binary code and libraries) and might come up with compilation errors or missing headers and the like. There's sadly no magic silver bullet solution for those, and ~Google~ DuckDuckGo and StackOverflow are usually your best friends here.
```

Comment:
```suggestion
> This also sadly comes with risks of having installation troubles depending on the environment of your machine (and which libraries and headers you have installed, etc); in fact, it's common to have issues when trying to `brew install imagemagick` or to install `rmagick`, especially on the steps which tries to compile those libraries (since they rely on compiled, binary code and libraries) and might come up with compilation errors or missing headers and the like. There's sadly no magic silver bullet solution for those, and search engines and StackOverflow are usually your best friends here.
```

- Nitpick: "sadly" already used at the start of the sentence
- Nitpick: removed double negation
- Nitpick: removed potential controversy on whether to pick Google or DDG